neo/io/micromedio.py

```diff
@@ -76,16 +71,16 @@ def read_segment(self, cascade=True, lazy=False, ):
         """
         Arguments:
         """
-        f = StructFile(self.filename, 'rb')
+        f = StructFile(open(self.filename, 'rb'))
```

Comment:
how does this actually work? Will there be an open file handle floating around? In that case using a with-statement could help.